@xen-orchestra/proxy/src/app/mixins/remotes.js

```diff
@@ -49,7 +49,7 @@ export default class Remotes {
   @decorateWith(deduped, remote => [remote.url])
   @decorateWith(disposable)
   async *getHandler(remote, options) {
```

Comment:
Is the `options` still necessary?

Reply:
To let the users of this method to override default options or to add more options.

Reply:
Is it used in current code?

If no, simply removes it for the moment :-)